### ISSUE TITLE
Pass `prefix_map` to implementation objects?

### ISSUE THREAD

**Issue by joeflack4** [INCATools/ontology-access-kit#236]
## Overview
This should be relevant in situations where (a) I want to consistently only CURIES back from `.entities()` (assuming all the term URI stems are contained in my `prefix_map`) and (b) when I want to get all of the terms '[owned' by the ontology](https://github.com/INCATools/ontology-access-kit/issues/220).

I'm so far mostly using `ProntoImplementation` and `SqlImplementation`, but I don't see a way to pass `prefix_map` to either of these on object initialization (or to the methods I'm interested in).

**Comment by matentzn** [comment]
I agree that both serialisation and deserialisation methods should have a prefix_map parameter.

**Comment by caufieldjh** [comment]
This would make sense in the context of planned updates for Biolink category maps too - it would be nice to be able to validate/transform to CURIEs and assign them categories at the same time.
